health/health.go

```diff
@@ -705,6 +706,12 @@ func (t *Tracker) SetIPNState(state string, wantRunning bool) {
 	t.mu.Lock()
 	defer t.mu.Unlock()
 	t.ipnState = state
+	if t.ipnStateFirstSet.IsZero() {
+		// The first time we see an IPNState getting set, it means the backend is starting up. We store this timestamp.
+		// We use it to silence some warnings that are expected during startup.
+		t.ipnStateFirstSet = time.Now()
+		t.setUnhealthyLocked(warmingUpWarnable, nil)
+	}
```

Comment:
I think we want to do this on any transition from wantRunning false to wantRunning true, not just once.

Reply:
Aargh, you're right. I'm used to our Apple platforms where there's always only one wantRunning for each process lifetime.